src/prog_algs/metrics/toe_profile_metrics.py

```diff
@@ -41,3 +43,42 @@ def alpha_lambda(toe_profile : ToEPredictionProfile, ground_truth : dict, lambda
                     print('\ttoe:', toe.key(key))
                     print('\tBounds: [{} - {}]({}%)'.format(lower_bound, upper_bound, toe.percentage_in_bounds([lower_bound, upper_bound])[key]))
             return result
+
+def prognostic_horizon(toe_profile : ToEPredictionProfile, criteria_eqn : Callable, ground_truth : dict, **kwargs):
+    """
+    Compute prognostic horizon metric, given by the difference between a time ti, when the predictions meet specified performance criteria, and the time corresponding to the end of life (EoL).
+    PH = EOL - ti
+    Args:
+        toe_profile (ToEPredictionProfile): A profile of predictions, the combination of multiple predictions
+        criteria_eqn (Callable function): A function calculating whether ground truth meets some criteria for corresponding key in ToEPredictionProfile
+        ground_truth (dict): Dictionary containing ground truth; specified as key, value pairs for event and its value
+        kwargs (optional): configuration arguments. Accepted args include:
+            * keys (list[string], optional): list of keys to use. If not provided, all keys are used.
+
+    Returns:
+        dict: Dictionary containing prognostic horizon calculations (value) for each event (key)
+    """
+    params = {
+        'print': False
+    }
+    params.update(kwargs)
+
+    ph_result = {k:False for k in ground_truth.keys()} # False means not yet met; will be either a numerical value or None if met
```

Comment:
```suggestion
    ph_result = {k:None for k in ground_truth.keys()} # False means not yet met; will be either a numerical value or None if met
```
Why not start with None and change the comparisons below to "is None" or "is not None"

Reply:
Then we should be able to remove lines 80-82